geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/WanLocatorDiscovererImpl.java

```diff
@@ -113,10 +122,17 @@ private RemoteLocatorJoinRequest buildRemoteDSJoinRequest(int port, Distribution
       final String hostnameForClients) {
     String localLocator = config.getStartLocator();
     DistributionLocatorId locatorId = null;
+
+    InternalDistributedSystem system = InternalDistributedSystem.getConnectedInstance();
```

Comment:
same comment about this method and the config parameter

Reply:
Thanks for comments. Updated.